lib/stats/stats-cluster-logpipe.c

```diff
@@ -51,7 +51,12 @@ _counter_group_logpipe_get_type_label(StatsCounterGroup *self, gint type, StatsC
   if (type >= SC_TYPE_MAX)
     return FALSE;
 
-  *label = stats_cluster_label("result", tag_names[type]);
+  const gchar *type_name = tag_names[type];
+
+  if (type == SC_TYPE_WRITTEN)
+    type_name = "delivered";
+
+  *label = stats_cluster_label("result", type_name);
```

Comment:
[optional]

I think it would be better to grab this from the other side, and rename `SC_TYPE_WRITTEN` and its corresponding format string, driver level `StatsCounterItem` names, etc, as this implementation suggests, that the legacy naming is the real one and the new-style has a little workaround for it. In other words, this implementation makes it harder to remove the legacy stats.

Reply:
You're right. I'd like to fix this separately. This PR became huge.

Reply:
Sure!